mojaloop/values.yaml

```diff
@@ -3483,6 +3485,9 @@ simulator:
 
   affinity: {}
 
+settlement:
+  enabled: true
```

Comment:
Why was this added? I believe the settlement chart is already enabbled?

Reply:
This is a new service, that handles settlements for the portal. I renamed the service into `portal-settlement`. Sorry for the confusion.